clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -301,6 +301,24 @@ public void testInvalidKeystorePassword() throws Exception {
             // Expected exception
         }
     }
+
+    /**
+     * Tests that client connections can be created to a server
+     * if null truststore password is used
+     */
+    @Test
+    public void testNullTruststorePassword() throws Exception {
```

Comment:
I understand that. My point was if you have a typo in the config name. In that case, you'll end up with a null password and integrity checking will be disabled even if the truststore has a password (in this test, the truststore password is set when we save it in `TestSslUtils.createSslConfig`). I found that a little surprising, I thought this was only about the case when you didn't want to set a password in the truststore.

In any case, you'll at least get a `unused config` warning and that should hopefully be a clue.